dfinity/nns-dapp
# GIX-1777: Format age bonus as percentage

# Motivation

Consistency with the dashboard and the new neuron details UI.

In this PR: Show age and dissolve delay bonus as percentage.

# Changes

* New components DissolveDelayBonusText and AgeBonusText.
* Use new components in related ActionItems.
* Add `testId` prop to Tooltip.

# Tests

* Test file and PO for new components DissolveDelayBonusText and AgeBonusText.
* Update test cases to expect percentage.
* Use new POs in the ActioItem POs.

# Todos

Covered with the entry of the new neuron details UI.
